components/eam/cime_config/config_compsets.xml

```diff
@@ -133,27 +133,27 @@
 
   <compset>
     <alias>FSCM-ARM97-MMF1</alias>
-    <lname>ARM97_EAM%SCM-MMF1_ELM%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
+    <lname>ARM97_EAM%SCM-MMF1_ELM%SP_SICE_DOCN%DOM_SROF_SGLC_SWAV</lname>
```

Comment:
The equivalent of CICE%PRES is MPASSI%PRES.   SICE is the stub sea ice model and I think would mean there is no sea ice at all in the arctic or antarctic.  Might run but would not be Earth.

Reply:
Yea, but we don't need it for this compset since it's a single column over land, so I figured we don't want to bother building MPAS-SI if we don't need to. Even for the other SCM RICO compset, the column is at a low enough latitude that we don't have to worry about sea ice.

Reply:
I see you used MPASSI below. Did you intend SICE here?

Reply:
Yes, because it's SCM over land like I mentioned above.